src/index.js

```diff
@@ -99,7 +100,19 @@ const updateDailySummaryTable = async (sprint, pointsLeft) => {
   });
 };
 
-const getPointsLeftByDay = async (sprint, start) => {
+const getChartDatasets = async (sprint, start, includedEnd) => {
```

Comment:
Hmm can we name this better so it is less vague?

Reply:
Also what does `includedEnd` mean?

Reply:
any name suggestions? I was trying to think of something that includes the guideline and the data

Reply:
the included end was because i needed to push `end` back by one day to account for the +1 to the length. So `includedEnd` is the end date (inclusive) and `end` date is the end date (exclusive). Explained in a bit more detail in lines 104-110